CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
+- Add: servicePath field to builtin attributes, so return ServicePath in the entity response (#2877)
```

Comment:
```suggestion
- Add: servicePath field to builtin attributes (#2877)
```

No need of the last part (in addition, it is uncompleted, as the new builtin attribute is not only for entity responses, but for subscription notifications).

We describe it properly in documentation. I'll do that PR.

Reply:
That PR is https://github.com/telefonicaid/fiware-orion/pull/4380

Reply:
Resolve in c3eec37